### PR TITLE
Remove dependency on polymer iron-icon and mwc-icon, use sc-icon.js

### PR DESCRIPTION
**Pull request by sujato**
This branch fixes the issue here

https://github.com/suttacentral/suttacentral/issues/1903

All the icons are replaced with a new file, `sc-icon.js`. Don't use `<iron-icon>` or `<mwc-icon>` at all. Instead use lit-html's `svg` method.

I also made a number of other changes: 

- deleting unused code (esp. the tooltip code from simple-text page. There is no tooltip any more).
- remove all ripples from suttaplex.

## Don't merge until the following are all fixed

<s>Some things I couldn't figure out how to do, so I made them fail disgracefully. Hopefully the intent should be clear. These are all cases where the icons are shown conditionally, and I don't know how to write the logic.</s>

These all now appear to be fixed. The only one I can't test is the error icons, let's hope they work!

- [x]  `sc-suttaplex.js`:  `difficulty` icons
- [ ]  `sc-error-icon.js`
- [x]  `sc-parallel-list`:  the `parallels-relation-cell` icons have to be computed

In one cases I can't fix them because the page isn't there yet. But don't worry, merge anyway and check later.

- `offline-page`: still uses iron-icons

### CODE REVIEW

File: client/img/sc-icon.js [hunk 1]
```diff
@@ -0,0 +1,63 @@
...
+info_outline: svg`<svg class="info_outline icon" xmlns="http://www.w3.org/2000/svg" height="24" viewBox="0 0 24 24" width="24"><path d="M11 17h2v-6h-2v6zm1-15C6.48 2 2 6.48 2 12s4.48 10 10 10 10-4.48 10-10S17.52 2 12 2zm0 18c-4.41 0-8-3.59-8-8s3.59-8 8-8 8 3.59 8 8-3.59 8-8 8zM11 9h2V7h-2v2z"></path></svg>`,
+intermediate: svg`<svg xmlns="http://www.w3.org/2000/svg" class="intermediate icon" viewBox="0 0 100 125"><path d="M25.8 54.9c-.6 0-1 .4-1 1v10.3c0 .6.4 1 1 1H29l5.7 25.2c.1.5.5.8 1 .8h28.7a1 1 0 001-.8L71 67.2h3.2c.6 0 1-.4 1-1V55.9c0-.6-.4-1-1-1H51v-26l2.5-.1a18 18 0 0017.4-17.4l.1-3.5a1 1 0 00-.3-.7 1 1 0 00-.7-.3l-3.5.1a18 18 0 00-17.4 17.4l-.1 3.5v4.5a18 18 0 00-15.6-10l-3.5-.1a1 1 0 00-.7.3 1 1 0 00-.3.7l.1 3.5a18 18 0 0017.4 17.4l2.5.1v10.5H25.8zm20.7-12.6A16.1 16.1 0 0131 26.8l-.1-2.5 2.5.1c8.4.3 15.2 7.1 15.5 15.5l.1 2.5-2.5-.1zm4.6-17.9c.3-8.4 7.1-15.2 15.5-15.5l2.5-.1-.1 2.5a16.1 16.1 0 01-15.5 15.5l-2.5.1.1-2.5zm12.4 66.8h-27l-5.4-24H69l-5.5 24zm9.7-26H26.8v-8.3h46.3v8.3z"/></svg>`,
```
Comment (Prograd) [inline client/img/sc-icon.js:30]: This icon doesn't work. Unfortunately I don't know how I can get a proper one.

Comment (Prograd) [reply client/img/sc-icon.js:30]: I fixed the difficulty icon in 26e7750.

**Review by ihongda — COMMENTED** [review]
Running locally looks like everything is fine :tada: , and it would be better if the code was formatted as standard :smile:

**Comment by sujato** [comment]
I think the code formatting should be taken care of by linting when Robert adds that, so let's go ahead and merge.

**Pull request merged**